BrainPortal/app/controllers/tool_configs_controller.rb

```diff
@@ -167,7 +167,7 @@ def update #:nodoc:
     form_tool_config.bourreau_id = @tool_config.bourreau_id
 
     # Update everything else
-    [ :version_name, :description, :script_prologue, :group_id, :ncpus, :docker_image,
+    [ :version_name, :description, :script_prologue, :group_id, :ncpus, :docker_image, :singularity_image_userfile_id,
```

Comment:
It is normal. The support for Docker pull a `docker_image` from dockerhub, and the support for Singularity  use a `singularity_image_userfile_id` registered in CBRAIN.